### PR TITLE
Simplify the overriding of Trepn preferences

### PR DESCRIPTION
**Pull request by sshann**
This commit extends the overriding logic of Trepn preferences into a self-contained function, allowing users to override all configurations. Mirror the preferences key in the user-defined configuration to extend the overriding to any preference. To reduce the verbosity of the key, the mirrored key contains only the substring after the last dot. For instance, the preference `com.quicinc.preferences.general.profiling_interval` is mapped into `profiling_interval`.

### CODE REVIEW

File: AndroidRunner/Plugins/trepn/Trepn.py [hunk 1]
```diff
@@ -166,7 +193,8 @@ def aggregate_end(self, data_dir, output_file):
 
     def aggregate_trepn_subject(self, logs_dir):
         def add_row(accum, new):
-            row = {key: value + float(new[key]) for key, value in list(accum.items()) if key not in ['Component', 'count']}
+            row = {key: value + float(new[key]) for key, value in list(accum.items()) if
+                   key not in ['Component', 'count']}
```
Comment (sshann) [inline AndroidRunner/Plugins/trepn/Trepn.py:197]: This and the change below were autoformatted due to the lines being so long.
If prefered, I can undo these changes.

**Comment by sshann** [comment]
@EricZielinski @iivanoo

It seems like I cannot tag you as reviewers, so I am mentioning you via the comments.

**Comment by sshann** [comment]
I am doing a small change to allow to automatically override the remaining preferences without further changes in the source code

**Comment by EricZielinski** [comment]
Looks good, but can you also update the relevant example configurations that use Trepn in `examples/`?

**Comment by sshann** [comment]
Native and web examples are updated.
I took the liberty to format the example files.

**Comment by sshann** [comment]
By the way, wouldn't it be nicer to have the Trepn and the Android profiler in their own directory?
I assume that they were the first and the original author decided to organize by native/web.
The profilers that were added afterwards are all in their own directory, so it makes sense to also update Trepn and Android to their own directory.

**Comment by EricZielinski** [comment]
> By the way, wouldn't it be nicer to have the Trepn and the Android profiler in their own directory?
> I assume that they were the first and the original author decided to organize by native/web.
> The profilers that were added afterwards are all in their own directory, so it makes sense to also update Trepn and Android to their own directory.

Yes, I think you're right - they were the first.  It makes sense to separate them.

**Pull request merged**